server.R

```diff
@@ -345,45 +356,26 @@ shinyServer(function(input, output, session) {
                             fluidRow(
                               column(
                                 width=2,
-                                tags$style(HTML('#NextQuestion{background-color:green}')),
-                                tags$style(HTML('#NextQuestion{color:white}')),
-                                tags$style(HTML('#NextQuestion{width:100%}')),
-                                actionButton("NextQuestion", "Next")
+                                tags$style(HTML('#BackButton{background-color:grey}')),
+                                tags$style(HTML('#BackButton{color:white}')),
+                                tags$style(HTML('#BackButton{width:100%}')),
+                                actionButton("BackButton", "Back") #changed to uk order
```

Comment:
"Change to UK" comment not necessary in the code, can just add this to the commit messages.